cores/nRF5/wiring_digital.c

```diff
@@ -86,18 +93,42 @@ void digitalWrite( uint32_t ulPin, uint32_t ulVal )
 
   ulPin = g_ADigitalPinMap[ulPin];
 
+  for (uint8_t i = 0; i < PWM_COUNT; i++)
```

Comment:
What do you think about moving this to a private `analogWriteOff` (or something with a better name) function in `cores/nRF5/wiring_analog_nRF5x.c`?

Reply:
Rather than creating another function I could change analogWrite so that it turns off PWM when a duty cycle of zero is passed?

Its up to what you prefer as you are the owner and this affects the API. Either implementation is easy to implement.